osu.Game/Overlays/MusicController.cs

```diff
@@ -243,6 +254,52 @@ private PreviousTrackResult prev()
                 onSuccess?.Invoke();
         });
 
+        /// <summary>
+        /// Attenuates the volume and/or filters the currently playing track.
+        /// </summary>
+        /// <param name="duration">Duration of the ducking transition, in ms.</param>
+        /// <param name="duckVolumeTo">Level to drop volume to (1.0 = 100%).</param>
+        /// <param name="duckCutoffTo">Cutoff frequency to drop `AudioFilter` to. Use `AudioFilter.MAX_LOWPASS_CUTOFF` to skip filter effect.</param>
```

Comment:
I'd probably prefer something more convenient like `null` to turn off the low pass, but not a huge deal I guess